src/modules/universal-discovery/components/create/create.component.js

```diff
@@ -6,15 +6,13 @@ import ChooseContentTypeComponent from './create.choose.content.type.component';
 
 import './css/create.component.css';
 
-export default class CreateComponent extends Component {
-    render() {
-        return (
-            <div className="c-create" style={{maxHeight:`${this.props.maxHeight}px`}}>
-                <ChooseLanguageComponent {...this.props} />
-                <ChooseContentTypeComponent {...this.props} />
-            </div>
-        );
-    }
+const CreateComponent = (props) => {
```

Comment:
You can drop import of React component in the first line: ` import React from 'react';`